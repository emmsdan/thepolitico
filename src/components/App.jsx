import React, { Component } from 'react';

import './App.scss';
import CallToAction from './presentationals/CallToAction/CallToAction';
import PartyList from './containers/PartyList';

class App extends Component {
  render() {
    return (
      <div>
        <CallToAction withButton={true} />
        <PartyList />
      </div>
    );
  }
}

export default App;
