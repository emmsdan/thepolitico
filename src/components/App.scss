@import "../assets/variables";

.toothy {
  color: $primary-black;
}
