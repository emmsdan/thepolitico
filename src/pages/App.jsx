import React, { Component } from 'react';

import './App.scss';
import CallToAction from '../components/presentationals/CallToAction/CallToAction';
import PartyList from '../components/containers/PartyList/PartyList';
import OfficeList from '../components/containers/OfficeList/OfficeList';

class App extends Component {
  render() {
    const headingStyled = {
      textAlign: 'center',
      fontSize: '30px',
      padding: '20px',
    };
    return (
      <div>
        <CallToAction withButton={true} />
        <h3 style={headingStyled}>Available Offices</h3>
        <OfficeList />
        <h3 style={headingStyled}>Politico Parties</h3>
        <PartyList />
      </div>
    );
  }
}

export default App;
