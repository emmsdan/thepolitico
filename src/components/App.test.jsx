import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  it('should render correctly', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
    expect(component.html()).toEqual(
      '<div><h1 class="toothy"> Welcome to the Politico. </h1></div>',
    );
  });
});
