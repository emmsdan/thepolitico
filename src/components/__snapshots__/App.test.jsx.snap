// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div>
  <h1
    className="toothy"
  >
     Welcome to the Politico. 
  </h1>
</div>
`;
